app/src/main/java/me/thuanc177/kotsune/libs/animeProvider/allanime/AllAnimeAPI.kt

```
@@ -0,0 +1,554 @@
+            val rawEpisodes = availableEpisodes?.optInt("raw", 0) ?: 0
+
+            // Determine episode count for requested translation type
+            val availableTypeEpisodes = when (translationType.lowercase()) {
+                "dub" -> dubEpisodes
+                "raw" -> rawEpisodes
+                else -> subEpisodes // Default to sub
+            }
+
+            val animeAnilistId = try {
+                result.getInt("aniListId")
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)